trail/query/rest_views.py

```diff
@@ -1,24 +1,45 @@
 import django
 from django.core.exceptions import FieldError
 
-import numpy as np
-
 from rest_framework import mixins, status
-from rest_framework import exceptions as rf_error
 from rest_framework.views import APIView
 from rest_framework.response import Response
 
 from upload.models import Metadata, Wcs
 from query.serializers import MetadataSerializer, WcsSerializer
 
-from drf_yasg.utils import swagger_auto_schema
-
 
 class GenericTableView(mixins.ListModelMixin, APIView):
+    """
+     GenericTableView that serves as an interface to provide one unify rest api query
+     to different tables
+    """
+
     queryset = None
     serializer_class = None
 
-    def get_shared_non_query_keys(self, queryDict, *args, **kwargs):
+    def get_shared_non_query_keys(self, queryDict):
+        """
+        Extracts the keys and values not directly related to the query from the given query dict,
+        and copies them into a queryset related parameter
+        and serializer related parameter dictionaries.
+        Extracts only those keys and values that are shared among all table views,
+        such as fields or casesensitive
+
+        Parameters
+        ----------
+        queryDict: django.QueryDict
+            The queryDict converted from user inputs
+
+        Returns
+        -------
+        serializerParams : Dictionary
+            The dictionary containing all serializer related param specified by each table.
+        querysetParams : Dictionary
```

Comment:
dictionary --> `dict`

In general, you put the same names here as you would if you were in a python repl. So if you would declare a new dictionary with `dict()` you use dict, if you use a list then it's `list` if you use some class from some other module then it's `module.submodule.class`